docs/sources/flow/reference/components/otelcol.exporter.jaeger.md

```diff
@@ -7,9 +7,11 @@ title: otelcol.exporter.jaeger
 `otelcol.exporter.jaeger` accepts telemetry data from other `otelcol` components
 and writes them over the network using the Jaeger protocol.
 
-> **NOTE**: `otelcol.exporter.jaeger` is a wrapper over the upstream
-> OpenTelemetry Collector `jaeger` exporter. Bug reports or feature requests will
-> be redirected to the upstream repository, if necessary.
+> **NOTE**: `otelcol.exporter.jaeger` is a wrapper over the 
+> [upstream](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/jaegerexporter) 
+> OpenTelemetry Collector `jaeger` exporter. The upstream
+> exporter has been deprecated and will be removed from future versions of 
+> both OpenTelemetry Collector and Grafana Agent because Jaeger supports OTLP directly.
```

Comment:
We use the `> **NOTE**:` syntax in all the Flow Otel pages. I would be happy to change it, but it'd be best to do it for all pages at once so we are consistent. Would you like me to raise a separate issue for it? It would also be nice to know why the `{{% admonition type="note" %}}` syntax is preferable?

Reply:
The `> **NOTE**:` syntax formats the note as a blockquote. This works, but it relies on the blockquote formatting in the stylesheets staying constant and as expected. 

The shortcode `{{% admonition type="note" %}}` ensures that ALL notes are formatted the same regardless of where they are in the docs... same with warnings and cautions... and the stylesheet is easier to manage.

Refer also to https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition-shortcode for formatting of the admonition shortcode.

So yes, if it's in the template we should update it.

Reply:
If we leave it as-is in this PR, that's fine. We should plan to update the template for future